Source/Scene/Model.js

```diff
@@ -156,8 +156,49 @@ define([
         this.skinnedNodesIds = [];
     }
 
+    LoadResources.prototype.getBufferBytes = function(buffer, byteOffset, byteLength) {
+        return getSubarray(buffer, byteOffset, byteLength);
+    };
+
     LoadResources.prototype.getBuffer = function(bufferView) {
-        return getSubarray(this.buffers[bufferView.buffer], bufferView.byteOffset, bufferView.byteLength);
+        return this.getBufferBytes(this.buffers[bufferView.buffer], bufferView.byteOffset, bufferView.byteLength);
+    };
+
+    LoadResources.prototype.getDecodedBuffer = function(bufferView, decodeBufferAccessors) {
+        var bufferData = new Uint8Array(0);
+        var byteShift = 0;
+        for (var i in decodeBufferAccessors) {
+            var accessor = decodeBufferAccessors[i];
+            var chunkData = this.getBufferBytes(this.buffers[bufferView.buffer], bufferView.byteOffset + accessor.byteOffset, accessor.byteStride * accessor.count);
+            var extensions = accessor.extensions;
+            if (defined(extensions)) {
+                var attributes = extensions.WEB3D_quantized_attributes;
+                if (defined(attributes)) {
+                    var decodeMatrix = attributes.decodeMatrix;
```

Comment:
Are we decoding here?  This is supposed to happen on the GPU in the vertex shader so that we keep the data compressed and decode for free by burning this matrix into the model matrix.  That is the critical insight of this approach: not the best compression, but pretty good and free to decompress.

This will likely completely changed your implementation.

@bagnell can advise.